client/landing/stepper/declarative-flow/new-hosted-site-flow.ts

```diff
@@ -149,9 +147,6 @@ const hosting: Flow = {
 			() => {
 				if ( currentStepSlug === undefined ) {
 					resetOnboardStore();
-					if ( query.get( 'campaign' ) === 'reddit' ) {
-						reduxDispatch( savePreference( 'hosting-trial-campaign', 'reddit' ) );
-					}
```

Comment:
The Reddit campaign work overrides the A/B test to enable the free trial. Now that we’ll stop the A/B test, it doesn’t make sense to keep the Reddit campaign logic